cmd/test_runner_test.go

```diff
@@ -168,6 +169,15 @@ func (run *TestRunner) DatasetMarshalJSON(t *testing.T, ref string) (data string
 	return data
 }
 
+// Dscache returns the dscache from the repo
+func (run *TestRunner) Dscache(t *testing.T) *dscache.Dscache {
+	repo, err := run.RepoRoot.Repo()
+	if err != nil {
+		t.Fatal(err)
+	}
+	return repo.Dscache()
+}
```

Comment:
This is only used in one place, and its main point of failure is on Repo not existing. Do we really need it?